airbyte-integrations/connectors/destination-selectdb/src/main/resources/spec.json

```diff
@@ -0,0 +1,54 @@
+{
+  "documentationUrl": "https://docs.airbyte.com/integrations/destinations/selectdb",
+  "supportsIncremental": false,
+  "supportsNormalization": false,
+  "supportsDBT": false,
+  "supported_destination_sync_modes": ["append", "overwrite"],
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "SelectDB Destination Spec",
+    "type": "object",
+    "required": [
+      "load_url", "jdbc_url", "cluster_name", "user_name", "password", "database"
+    ],
+    "properties": {
+      "load_url": {
+        "title": "loadURL",
+        "description": "load host and port: xxx.privatelink.aliyun.com:47057",
+        "type": "string",
+        "order": 0
+      },
+      "jdbc_url": {
+        "title": "jdbcURL",
+        "description": "jdbc host and port: xxx.privatelink.aliyun.com:30523",
+        "type": "string",
+        "order": 1
+      },
+      "cluster_name": {
```

Comment:
do I need the cluster name that can be found here?
![image](https://user-images.githubusercontent.com/10071532/222008239-4f51ee40-0e8d-444c-8f7a-c88c41e8a5fd.png)

Reply:
yes